salesforce_api/mysql-record-to-sfdc-new-product/sample.bal

```diff
@@ -0,0 +1,52 @@
+import ballerinax/mysql;
+import ballerinax/salesforce as sfdc;
+
+type Product record {
+    string Name;
+    string Product_Unit__c;
+    string CurrencyIsoCode;
+};
+
+type ProductRecieved record {
+    string name;
+    string unitType;
+    string currencyISO;
+    string productId;
+};
+
+const int HEADINGS_ROW = 1;
+
+//mySQL configuration parameters
+configurable int port = ?;
+configurable string host = ?;
+configurable string user = ?;
+configurable string database = ?;
+configurable string password = ?;
+
+// Salesforce configuration parameters
+configurable string salesforceAccessToken = ?;
+configurable string salesforceBaseUrl = ?;
+
+sfdc:Client sfdcClient = check new ({
+    baseUrl: salesforceBaseUrl,
+    auth: {
+        token: salesforceAccessToken
+    }
+});
+
+public function main() returns error? {
+    mysql:Client dbClient = check new (host = host, user = user, password = password, database = database, port = port, options = {});
+
+    stream<ProductRecieved, error?> streamOutput = dbClient->query(`SELECT name, unitType, currencyISO, productId FROM products WHERE processed = false`);
```

Comment:
break the line at 120 chars

Reply:
Updated in [bcf09ac](https://github.com/ballerina-guides/integration-samples/pull/31/commits/bcf09ac42e87543485854586312facacacd050cb)